otherlibs/stdune/src/ansi_color.ml

```diff
@@ -4,13 +4,17 @@ module RGB8 : sig
   val to_dyn : t -> Dyn.t
   val of_int : int -> t
   val to_int : t -> int
+  val of_char : char -> t
+  val to_char : t -> char
```

Comment:
I was a little hesitant to expose these. Maybe I should put them in a module. It is very unlikely that the internal representation will change for these, but on the other hand I don't want to make the consumer of the API think that it is just a char.

Reply:
Seems fine to me.